src/server/terrain/biomes.zig

```diff
@@ -289,6 +289,7 @@ pub const Biome = struct { // MARK: Biome
 	caves: f32,
 	caveRadiusFactor: f32,
 	crystals: u32,
+	erosion: f32, // How much of the surface structure should be eroded depending on the slope.
```

Comment:
I think erosion is a term that's too much associated with large scale structures, such mountain ranges and rivers, but what you are doing here is more of a small-scale effect that doesn't (and shouldn't) impact the terrain shape at all.

I think soilCreep would be a more fitting and more specific term.